### DISCUSSION TITLE
Accept NineAnimator backup format

### DISCUSSION THREAD

**Discussion by raffaeledf9** [cranci1/Ryu#35] [Ideas]
Here I am with another suggestion. Since in these days NineAnimator is literally falling apart in pieces, I believe many people as discovering the existence of Ryu trying to find a solution. I don’t know how different they are, but since Ryu is largely based on NineAnimator, could it be possible to support its backup format? It would be so convenient for an easy and rapid migration of the saved content from it to Ryu. Thanks in advance.

**Comment by cranci1** [comment]
I did try looking in to the nineanimator backup file but its pretty much impossible to support them, mainly because datas are just completely different

**Comment by raffaeledf9** [comment]
Thanks anyway for trying!
